app/views/RoomView/index.js

```diff
@@ -452,6 +455,10 @@ class RoomView extends React.Component {
 		}
 	}
 
+	setFileLoading = (fileLoading) => {
```

Comment:
Why do we need this function?

Reply:
to start/stop loader

Reply:
I think you can use `setFileLoading = fileLoading => this.setState({ fileLoading });`

Reply:
😍 😍 😍